java/Nikiforov-Oleksandr/src/com/zoolatech/lecture1/tasks/_1/Task1.java

```diff
@@ -0,0 +1,22 @@
+package com.zoolatech.lecture1.tasks._1;
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts numbers of miles as a floating point number,
+ * converts it to kilometers and outputs the result.
+ */
+
+
+
+public class Task1 {
+    public static void main(String[] args) {
+        // task solution code here
+        Scanner sc = new Scanner(System.in);
+        milesToKm(sc.nextFloat());
+    }
+
+    public static void milesToKm(float distance){
+        // converts distance from miles to kilometers multiplied by 1.60934412
```

Comment:
you could remove the comment if you'd defined `distance * 1.60934412f` as some variable

Reply:
please, also add a variable